src/rebar_prv_xref.erl

```diff
@@ -301,6 +301,7 @@ find_function_source_in_abstract_code(F, A, AbstractCode) ->
                safe_element(3, E) == F,
                safe_element(4, E) == A],
     case Fn of
+        [{function, {Line, _Column}, F, _, _}] -> {Source, Line};
```

Comment:
To prevent Dialyzer balking, we should use https://erlang.org/doc/man/erl_anno.html#location-1 to extract a non-opaque location marker.